docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
 You can choose which tasks you want to see:
```

Comment:
Rephrase to 

"Click on the selection field in the left panel to choose which tasks you want to see."